<launch>
    <node name="TestWResponseControl" pkg="wresponsecontrol" type="WResponseControl">
        <rosparam file="$(find wresponsecontrol)/config/test.yaml"/>
    </node>
</launch>